File: src/main/java/io/naztech/mgmntui/entity/Org.java
package io.naztech.mgmntui.entity;

import ch.rasc.extclassgenerator.Model;
import lombok.Data;
import lombok.RequiredArgsConstructor;

@Model(value = "Starter.model.Org", identifier = "negative", rootProperty = "records",
		createMethod = "orgService.create", readMethod = "orgService.read",
		updateMethod = "orgService.update", destroyMethod = "orgService.destroy",
		autodetectTypes = false)

@Data
@RequiredArgsConstructor
public class Org{
	
	private String orgName;

	private int orgNumber;

	private int id;
}

File: client/app/model/Org.js
Ext.define("Starter.model.Org",{
	extend:"Ext.data.Model",
	requires : [ "Ext.data.identifier.Negative", "Ext.data.proxy.Direct", "Ext.data.validator.Presence" ],
	identifier : "negative",
	fields : [ "orgName",{
		name : "orgNumber"
	},"id" ],
	proxy : {
		type : "direct",
		api : {
			read : "OrgService.read",
			create : "OrgService.create",
			update : "OrgService.update",
			destroy : "OrgService.destroy"
		},
		reader : {
			rootProperty : "records"
		},
		writer : {
			writeAllFields : true
		}
	}
});